### PR TITLE
📝 docs: add readme badges

### LINKED ISSUE #51: 📝 Documentation: Add README.md badges for GitHub Actions, Markdownlint, Prettier, and TypeScript
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/LearningTypeScript/projects/issues?q=is%3Aissue+label%3A%22area%3A+tooling) and found none that matched my request.
- [X] This is the appropriate issue form for the tooling issue I would like to report.

### Expected

Right now the README.md only includes a badge for the all-contributors count.

### Actual

Let's add badges there! At the very least, the relevant tools right now are:

* GitHub Actions ("build passing")
* Markdownlint
* Prettier
* TypeScript

### Additional Info

See https://github.com/JoshuaKGoldberg/TypeStat for badge formatting. We can ignore the npm badge as this repo doesn't (yet?) publish to npm.

### PR DESCRIPTION
**Pull request by fuadop**
## PR Checklist

- [x] Addresses an existing open issue: fixes #51 
- [x] That issue was marked as [accepting prs](https://github.com/LearningTypeScript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/LearningTypeScript/projects/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

- Adds readme badges for github actions, markdownlint, prettier and typescript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+    <img alt="Code Style: Prettier" src="https://img.shields.io/badge/code_style-prettier-14cc21.svg">
+  </a>
+  <a href="https://github.com/LearningTypeScript/projects/actions/workflows/markdown.yml">
+    <img alt="Markdown Style: Markdownlint" src="https://img.shields.io/badge/markdown_style-markdownlint-14cc21.svg">
+  </a>
+  <a href="https://github.com/LearningTypeScript/projects/actions">
+    <img alt="Github Actions: Build Passing" src="https://img.shields.io/badge/github_actions-build_passing-14cc21.svg">
+  </a>
+  <img alt="TypeScript: Strict" src="https://img.shields.io/badge/typescript-strict-14cc21.svg">
+</p>
```
Comment (JoshuaKGoldberg) [inline README.md:15]: Hmm, these badges look great alone, but it's a little odd seeing them center-aligned when both the heading and existing all-contributors badge are left-aligned:

Screenshot of these new center-aligned badges between the left-aligned header and left-aligned existing all-contributors badge

I don't really have a preference for centered vs. left-aligned. So as long as everything goes with the same alignment, whichever way you like more / is less of a pain would work.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```
Comment (JoshuaKGoldberg) [inline README.md:19]: This badge is the only one that uses `style=flat-square`. Can you either use https://allcontributors.org/docs/en/bot/configuration to make this one the same style as the others, or make the others also flat? I also don't really have a preference for the style, so whichever you like more!

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
A great start, they look very snazzy! 🙌

Just requesting changes on consistency. Thanks!

File: README.md (hunk 1 above)
Comment (fuadop) [reply README.md:15]: I didn't want to touch the contributors badge.. since it stated not to modify 😬

Comment (JoshuaKGoldberg) [reply README.md:15]: Yeah those comments are pretty aggressive haha. I'm betting you can wrap _around_ them in some way... or failing that, `badgeTemplate` from https://allcontributors.org/docs/en/bot/configuration.

Comment (fuadop) [reply README.md:15]: https://github.com/all-contributors/all-contributors/issues/361

Comment (fuadop) [reply README.md:15]: oh looks like I have to edit the `badgeTemplate`. I think this repo did something cool, maybe it's variables in markdown or so.. https://github.com/squirrellyjs/squirrelly/blob/master/README.md

**Comment by fuadop** [comment]
I have made changes @JoshuaKGoldberg 🎉

File: .all-contributorsrc [hunk 3]
```diff
@@ -4,6 +4,7 @@
   ],
   "imageSize": 100,
   "commit": false,
+  "badgeTemplate": "[ac-badge]: https://img.shields.io/badge/all_contributors-<%= contributors.length %>-orange.svg?style=flat",
```
Comment (JoshuaKGoldberg) [inline .all-contributorsrc:7]: Nice, looks like this works! Just the one last note, everything else is color 14cc21 and not `style=flat`, so let's go with that?

```suggestion
  "badgeTemplate": "[ac-badge]: https://img.shields.io/badge/all_contributors-<%= contributors.length %>-14cc21.svg",
```

**Review by JoshuaKGoldberg — APPROVED** [review]
Looks great, thanks!

I've just got a couple of tweaks there but since you got the structure down, I'll go ahead and just merge them. 👍

**Comment by fuadop** [comment]
Thanks @JoshuaKGoldberg ✨

**Pull request merged**